Lib/test/test_import/__init__.py

```diff
@@ -1343,38 +1343,22 @@ def test_circular_from_import(self):
         )
 
     def test_unwritable_module(self):
-        package = inspect.cleandoc('''
-            import sys
-
-            class MyMod(object):
-               __slots__ = ['__builtins__', '__cached__', '__doc__',
-                            '__file__', '__loader__', '__name__',
-                            '__package__', '__path__', '__spec__']
-               def __init__(self):
-                   for attr in self.__slots__:
-                       setattr(self, attr, globals()[attr])
-
-
-            sys.modules['spam'] = MyMod()
-        ''')
-
-        init = os.path.join('spam', '__init__')
-        with _ready_to_import(init, package) as (name, path):
-            with open(os.path.join(os.path.dirname(path), 'x.py'), 'w+'):
-                pass
-
+        sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'data'))
```

Comment:
Why the path insertion instead of importing `test.test_import.data.unwritable`?

Reply:
I was actually just following the pattern I saw in test_concurrency but this is certainly much cleaner!